lib/rules/alpha-value-notation/index.js

```diff
@@ -18,8 +18,8 @@ const messages = ruleMessages(ruleName, {
 	expected: (unfixed, fixed) => `Expected "${unfixed}" to be "${fixed}"`,
 });
 
-const ALPHA_PROPS = ['opacity', 'shape-image-threshold'];
-const ALPHA_FUNCS = ['hsl', 'hsla', 'hwb', 'lab', 'lch', 'rgb', 'rgba'];
+const ALPHA_PROPS = new Set(['opacity', 'shape-image-threshold']);
+const ALPHA_FUNCS = new Set(['hsl', 'hsla', 'hwb', 'lab', 'lch', 'rgb', 'rgba']);
```

Comment:
What is advantage of this over array?

Reply:
* https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prefer-set-has.md
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

You can find plenty of other references on Google